Formula/lua.rb

```diff
@@ -71,34 +44,35 @@ def install
     (lib/"pkgconfig/lua.pc").write pc_file
 
     # Fix some software potentially hunting for different pc names.
-    bin.install_symlink "lua" => "lua5.2"
-    bin.install_symlink "lua" => "lua-5.2"
-    bin.install_symlink "luac" => "luac5.2"
-    bin.install_symlink "luac" => "luac-5.2"
-    (include/"lua5.2").install_symlink include.children
-    (lib/"pkgconfig").install_symlink "lua.pc" => "lua5.2.pc"
-    (lib/"pkgconfig").install_symlink "lua.pc" => "lua-5.2.pc"
+    bin.install_symlink "lua" => "lua5.3"
+    bin.install_symlink "lua" => "lua-5.3"
+    bin.install_symlink "luac" => "luac5.3"
+    bin.install_symlink "luac" => "luac-5.3"
+    (include/"lua5.3").install_symlink include.children
+    (lib/"pkgconfig").install_symlink "lua.pc" => "lua5.3.pc"
+    (lib/"pkgconfig").install_symlink "lua.pc" => "lua-5.3.pc"
 
     # This resource must be handled after the main install, since there's a lua dep.
     # Keeping it in install rather than postinstall means we can bottle.
     if build.with? "luarocks"
       resource("luarocks").stage do
         ENV.prepend_path "PATH", bin
+        lua_prefix = prefix
```

Comment:
I'm honestly not sure why I added this in the first place given it was only used once, as far as I can see 😓.